fe/fe-core/src/main/java/com/starrocks/catalog/ExprPartitionMeta.java

```diff
@@ -0,0 +1,60 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.starrocks.catalog;
+
+import com.google.gson.annotations.SerializedName;
+import com.starrocks.analysis.Expr;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.List;
+
+public class ExprPartitionMeta {
+
+    private static final Logger LOG = LogManager.getLogger(ExpressionRangePartitionInfo.class);
+
+    @SerializedName(value = "partitionExprs")
+    private List<Expr> partitionExprs;
```

Comment:
Now Expr serialization is based on toSql(), which should be optimized before storing more complicated expr.

Reply:
use gson to serialization Expr

Reply:
ok